src/modules/commander/HomePosition.hpp

```diff
@@ -75,4 +80,10 @@ class HomePosition
 	uint8_t							_heading_reset_counter{0};
 	bool							_valid{false};
 	const failsafe_flags_s					&_failsafe_flags;
+	bool							_gps_position_for_home_valid{false};
+	double							_gps_lat{0};
```

Comment:
We could almost change the GPS sub to `SubscriptionData`

Reply:
Like 72260d00866d0aa39aa20ee434e947b13bfc2201?

Reply:
Reverted https://github.com/PX4/PX4-Autopilot/commit/72260d00866d0aa39aa20ee434e947b13bfc2201 again.